Argo/Globals/maybe.swift

```diff
@@ -0,0 +1,6 @@
+public func maybe<A, B>(b: B, f: A -> B, a: A?) -> B {
```

Comment:
hmm ya i guess that's all it is ... weird how that appeared after I broke down the types.